crates/router/src/connector/worldpay/transformers.rs

```diff
@@ -144,7 +144,24 @@ impl TryFrom<&types::ConnectorAuthType> for WorldpayAuthType {
                     api_key: Secret::new(auth_header),
                 })
             }
-            _ => Err(errors::ConnectorError::FailedToObtainAuthType)?,
+            types::ConnectorAuthType::CurrencyAuthKey { auth_key_map: _ } => {
```

Comment:
Hi @sudharsangs, 
Can you please revert back this change as wildcard match arms (_) is not in scope for AuthType?

Reply:
I've made the change @prasunna09